src/main/java/se/iths/librarysystem/entity/RoomEntity.java

```diff
@@ -10,9 +12,13 @@
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
     private Long id;
+
+    @NotBlank(message = "Room name is a required field")
     private String name;
     private int groupSize;
     private boolean internetAccess;
+
+    @NotEmpty(message = "Wheel chair access is a required field")
```

Comment:
I am getting an error when creating a room without `wheelchairAccess`, _"No validator could be found for constraint 'javax.validation.constraints.NotEmpty' validating type 'java.lang.Boolean'"_. The validator does not accept `@NotEmpty` constraint for a boolean. 
I have  tried `@NotBlank` and I get the same error. I have also tried `@NotNull` but this constraint has no effect even when `wheelchairAccess` is missing - I suspect this is because the default value for a boolean is already false.

Reply:
That makes sense. I will look further into it.

Reply:
Just going to remove the requirement. It has a default value and is functional.